server/api/src/rsGeneralAdmin.cpp

```diff
@@ -1135,9 +1138,29 @@ _rsGeneralAdmin( rsComm_t *rsComm, generalAdminInp_t *generalAdminInp ) {
     }
     if ( strcmp( generalAdminInp->arg0, "rm" ) == 0 ) {
         if ( strcmp( generalAdminInp->arg1, "user" ) == 0 ) {
+            const auto user_name = generalAdminInp->arg2 ? generalAdminInp->arg2 : "";
+            const auto zone_name = generalAdminInp->arg3 ? generalAdminInp->arg3 : "";
+            namespace iua = irods::experimental::administration;
+
+            if(!iua::server::exists(*rsComm, iua::user { user_name, zone_name } )) {
+                return CAT_INVALID_USER;
```

Comment:
Let's log a message for the admin.

_This applies to the group logic below too._